robot/EDA/tests/browser/settings/eda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -21,68 +27,37 @@ Validate Edit Mode For Accounts and Contacts Settings
     Wait for Locator                account_types.edit
     Click on Element                account_types.edit
 
-    # Check everything   
-    FOR    ${account}     IN      @{account_types}
-            Wait for Locator        ${account}
-            Click on Element        ${account}
-    END  
-
-'''
     Wait for Locator                account_types.administrative
     Click on Element                account_types.administrative
 
-    Wait for Locator                account_types.academic
-    Click on Element                account_types.academic
-
-    Wait for Locator                account_types.administrativecheckbox
-    Click on Element                account_types.administrativecheckbox
-
-    Wait for Locator                account_types.business
-    Click on Element                account_types.business
-
-    Wait for Locator                account_types.educational 
-    Click on Element                account_types.educational
-
-    Wait for Locator                account_types.household 
-    Click on Element                account_types.household
-
-    Wait for Locator                account_types.sports
-    Click on Element                account_types.sports
-
-    Wait for Locator                account_types.university 
-    Click on Element                account_types.university 
-'''
- 
-    # Click Button                  Save (in page object for this test)
+Click Checkboxes
+    Test Checkboxes                 Academic Program
+    Test Checkboxes                 Administrative
```

Comment:
A minor quibble: since you're using this keyword to check a single checkbox now that you aren't using a template, it might make more sense to call this "Test checkbox" (singular)

Otherwise, this test is definitely a big improvement in readability over the original version. 👍